### DISCUSSION TITLE
Relocate the cluster-operator to another Namespace

### DISCUSSION THREAD

**Discussion by itaydj** [strimzi/strimzi-kafka-operator#5468] [Q&A]
Hi,
We installed the Strimzi Cluster operator on a specific namespace and would like to relocate it to a new namespace in the same cluster.
Is there a way of doing that without affecting the already deployed resources (clusters, topics, users)?
We used helm to deploy it BTW.

Thanks

**Comment by scholzj** [comment]
I have no idea about Helm, so I can only suggest how to do it outside of Helm and maybe you or some other Helm user figure out how to do it with Helm.

The custom resource and the things they created are dependent on the Custom Resource Definitions. So when moving the operator to another namespace, it is important to delete only operator Deployment and the related parts (logging Config Map, RBAC resources), but not the CRDs. And then in the new namespace you just have to create the deleted resources again. So when installing from YAMLs, it is relatively simple operation.

**Reply by itaydj** [reply]
I understand, that should work, thanks you very much

**Comment by itaydj** [comment]
Hi,
for those installing the operator using Helm.
I wanted to updated that I just executed `helm uninstall strimzi` on the old namespace and than reinstalled it in the new one.
it did not remove any of the already installed resources.
